c/rowindex_array.cc

```diff
@@ -403,7 +403,7 @@ void ArrayRowIndexImpl::init_from_integer_column(const Column* col) {
   }
   int64_t imin = col->min_int64();
   int64_t imax = col->max_int64();
-  if (imin < 0) {
+  if (imin < -1) {
```

Comment:
So `imin` is allowed to be `-1`?

Reply:
Ah, this seems to be connected to #1496?

Reply:
yes